mockgcp/mocksql/sqluser.go

```diff
@@ -93,13 +93,6 @@ func (s *sqlUsersService) Insert(ctx context.Context, req *pb.SqlUsersInsertRequ
 	obj.Instance = name.Instance
 	obj.Kind = "sql#user"
 
-	if obj.PasswordPolicy == nil {
-		obj.PasswordPolicy = &pb.UserPasswordValidationPolicy{}
-	}
-	if obj.PasswordPolicy.Status == nil {
-		obj.PasswordPolicy.Status = &pb.PasswordStatus{}
-	}
-
```

Comment:
mind explaining why removing this ?

Reply:
For postgres + sqlserver, these fields are not sent in the GCP responses, so it felt "more correct" to remove it.

Reply:
Good to know! Thank you for the context!